src/Text/Pandoc/Writers/FB2.hs

```diff
@@ -96,7 +96,10 @@ pandocToFB2 opts (Pandoc meta blocks) = do
 
 description :: PandocMonad m => Meta -> FBM m Content
 description meta' = do
-  let genre = el "genre" "unrecognised"
+  let genre = case lookupMeta "genre" meta' of
```

Comment:
simplify this by using `lookupMetaString` from T.P.Writers.Shared.
Otherwise looks good!

Reply:
Done!